p2p/discover/topicindex/iptree.go

```diff
@@ -123,7 +130,7 @@ func (it *ipTree) computeScore(sum int, additional int) float64 {
 	if c == 0 {
```

Comment:
I'm not sure why is the `additional` parameter needed here.

Reply:
score measures the diversity of an ip address (considered to be added) against the current ip table state, so I don't think it makes sense to "compute the score that the addition of an IP would return". I think we can remove this additional parameter as michal did.